e2e/cypress/integration/messaging/file_upload_in_center_channel_spec.js

```diff
@@ -27,13 +27,13 @@ describe('Messaging', () => {
         cy.apiSavePreviewCollapsedPreference('false');
     });
 
-    it('M16425 : Show single image thumbnails in standard mode', () => {
+    it.only('M16425 : Show single image thumbnails in standard mode', () => {
```

Comment:
Remove `.only`

Reply:
Thanks for catching. Should be good now.